src/config/nunjucks/context/context.test.js

```diff
@@ -47,7 +47,7 @@ describe('#context', () => {
             url: '/about'
           }
         ],
-        serviceName: 'CDP Node.js Frontend Template',
+        serviceName: 'cdp-node-frontend-template',
         serviceUrl: '/'
```

Comment:
these values get substituted by the repo name, so was confusing to have the title name here